CMakeLists.txt

```diff
@@ -62,45 +43,5 @@ include_directories(
 add_definitions(${OPTSCHED_EXTRA_DEFINITIONS})
 link_directories(${OPTSCHED_EXTRA_LINK_LIBRARIES})
 
-if(LLVM_VERSION VERSION_LESS 7.0)
-  add_definitions(-DLLVM_DEBUG=DEBUG)
-endif()
```

Comment:
Are we dropping LLVM <7.0 support yet? If not, don't delete this.

Reply:
Do we have a reason to support LLVM < 7.0 ? The effect of this patch is to drop support for < 13.0

Reply:
That's fine then.